h2o-core/src/main/java/hex/grid/HyperSpaceWalker.java

```diff
@@ -464,7 +453,7 @@ public RandomDiscreteValueWalker(MP params,
     /** Based on the last model, the given array of ScoringInfo, and our stopping criteria should we stop early? */
     @Override
     public boolean stopEarly(Model model, ScoringInfo[] sk) {
-      return ScoreKeeper.stopEarly(ScoringInfo.scoreKeepers(sk),
+      return model != null && ScoreKeeper.stopEarly(ScoringInfo.scoreKeepers(sk),
```

Comment:
I would split this into multiple lines since now the args are not alligned with the method call

Reply:
@honzasterba see your point. Let's see if you will like new version.